### ISSUE TITLE
[Feature Request] Screen reader compatibility

### ISSUE THREAD

**Issue by unDeadHerbs** [benawad/vsinder#132]
When using a screen reader most of the buttons are unlabeled and some invisible regions are selectable.

As a quick example:  Looking at the card stack screen.
- The three tabs are unnamed.
- The left and right half of the picture are selectable, this is redundant with the buttons at the bottom.
- The code is not selectable and thus cannot be read.
   - I also suspect the text of the code is not attached to the image for reading.
- The card of the next user is selectable.
- The "Like" and "Not Like" buttons have no readable name.
